public/app/features/explore/utils/links.ts

```diff
@@ -23,7 +24,7 @@ import { splitOpen } from '../state/main';
 export const getFieldLinksForExplore = (options: {
   field: Field;
   rowIndex: number;
-  splitOpenFn?: typeof splitOpen;
+  splitOpenFn?: (options?: { datasourceUid: string; query: DataQuery; range?: TimeRange } | undefined) => void;
   range: TimeRange;
```

Comment:
See https://raintank-corp.slack.com/archives/CHKLEJ668/p1611251652003600 for more context on this.

Reply:
Maybe it would make sense to create a type for this and use it everywhere?

Reply:
Yes, I agree.

Reply:
I added a context for why I think this is wrong https://raintank-corp.slack.com/archives/CHKLEJ668/p1611681902018200?thread_ts=1611251652.003600&cid=CHKLEJ668.